.github/workflows/pr-sandbox.yaml

```diff
@@ -1,11 +1,7 @@
 name: Pull Request Sandbox
 # change to trigger a PR
 on:
-  pull_request:
-    paths:
-      - ".github/workflows/pr-sandbox.yaml"
-      - ".github/workflows/tf-plan-sandbox.yaml"
-      - "terraform/sandbox/**"
+  workflow_dispatch:
```

Comment:
This shouldn't be changing, this makes it a manual workflow and not triggered on PR

Reply:
Done.